src/win32.c

```diff
@@ -794,7 +795,14 @@ void win32_open_browser(const gchar *uri)
 				uri++;
 		}
 	}
-	ShellExecute(NULL, "open", uri, NULL, NULL, SW_SHOWNORMAL);
+	ret = (gint) ShellExecute(NULL, "open", uri, NULL, NULL, SW_SHOWNORMAL);
+	if (ret <= 32)
+	{
+		gchar *err = g_win32_error_message(GetLastError());
+		/* TODO add a GUI warning that opening an URI failed */
+		g_warning("ShellExecute failed opening \"%s\" (code %d): %s", uri, ret, err);
```

Comment:
You could add a translatable string if you want, it wouldn't get translated for next release but that wouldn't be worse than not translatable at all.  But well, if it's meant to be changed right after the release (would have to remember :)), I guess it doesn't really matter.